### DISCUSSION TITLE
Issue with pairing IKEA on/off E1723 switch

### DISCUSSION THREAD

**Discussion by diarmaidocualain** [Koenkk/zigbee2mqtt#5853] [Devices]
I am having difficulty in pairing the Tradfri on/off switch (E1743). It begins to pair, but fails the interview process. I was wondering if anyone has had a similar issue, and any suggested solutions? My set-up is as follows:

* Home Assistant
* zigbee2mqtt addon (1.17.0) I have also tried zigbee2mqtt and zigbee2mqtt-edge (version 1.17.0-dev (commit #4e2e04e)).
* CC2531 USB stick with firmware 20190608
* All running on Ubuntu 20.04 on an Intel NUC.

Pairing works with the Ikea motion sensor and Philips dimmer switch. My pairing process is to press the pairing button on the remote four times, and then immediately click on "scan" under Touchlink in the zigbee2mqtt UI. I have tried rebooting the host multiple times, and unplugging and plugging in the zigbee USB stick. After the interview fails, a device listed as "unsupported" manufacturer is given under the "Devices" tab. I have tried a "force remove" of this device and another pairing to no joy. I have tried to keep the remote awake by slowly tapping either one of the on/off buttons or the pairing buttons to no joy either. In addition, I have tried clicking on Touchlink "scan" at various times during the remotes pairing mode. 

```
Zigbee2MQTT:info  2021-01-21 14:29:36: Start Touchlink scan
Zigbee2MQTT:info  2021-01-21 14:29:47: Finished Touchlink scan
Zigbee2MQTT:info  2021-01-21 14:29:47: MQTT publish: topic 'zigbee2mqtt/bridge/response/touchlink/scan', payload '{"data":{"found":[]},"status":"ok","transaction":"amq0c-2"}'
Zigbee2MQTT:info  2021-01-21 14:29:56: Device '0x5c0272fffe264810' joined
Zigbee2MQTT:info  2021-01-21 14:29:56: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x5c0272fffe264810","ieee_address":"0x5c0272fffe264810"},"type":"device_joined"}'
Zigbee2MQTT:info  2021-01-21 14:29:56: Starting interview of '0x5c0272fffe264810'
Zigbee2MQTT:info  2021-01-21 14:29:56: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x5c0272fffe264810","ieee_address":"0x5c0272fffe264810","status":"started"},"type":"device_interview"}'
Zigbee2MQTT:info  2021-01-21 14:29:56: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0x5c0272fffe264810"},"type":"device_connected"}'
Zigbee2MQTT:info  2021-01-21 14:29:56: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0x5c0272fffe264810"},"type":"pairing"}'
Zigbee2MQTT:error 2021-01-21 14:32:14: Failed to interview '0x5c0272fffe264810', device has not successfully been paired
Zigbee2MQTT:info  2021-01-21 14:32:14: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x5c0272fffe264810","ieee_address":"0x5c0272fffe264810","status":"failed"},"type":"device_interview"}'
Zigbee2MQTT:info  2021-01-21 14:32:14: MQTT publish: topic 'zigbee2mqtt/bridge/devices', payload 
...
,"network_address":63559,"power_source":"Battery","software_build_id":"2.0.022","supported":true,"type":"EndDevice"},{"definition":null,"endpoints":{},"friendly_name":"0x5c0272fffe264810","ieee_address":"0x5c0272fffe264810","interview_completed":false,"interviewing":false,"network_address":32172,"supported":false,"type":"Unknown"}]'
Zigbee2MQTT:info  2021-01-21 14:32:14: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_failed","meta":{"friendly_name":"0x5c0272fffe264810"},"type":"pairing"}'
```

Thanks!

**Comment by Koenkk** [comment]
You shouldn't do the touchlink scan, this will guarantee a interview failure because the coordinator switches channel during this.

Check: https://www.zigbee2mqtt.io/information/FAQ.html#interview-fails especially: If it’s a battery powered device, try replacing the battery.

**Comment by m0wlheld** [comment]
> My pairing process is to press the pairing button on the remote four times, and then immediately click on "scan" under Touchlink in the zigbee2mqtt UI.

my pairing process is to set permit_join to true via MQTT publish, then push the pairing button of the switch 4 times. The red light on the button‘s front starts to pulse and after 20-40 secs pairing is done.

**Comment by diarmaidocualain** [comment]
Thanks for the tips. I am after successfully pairing two of my IKEA on/off remotes now. It looks like my issue was as you said, that clicking on the Touchlink button was breaking the pairing process. Once I stopped doing that, they linked after one or two attempts.

Thanks! :-)

**Comment by GryKyo** [comment]
I have two of these IKEA buttons.  One of these paired immediately with zigbee2mqtt running on a Raspberry Pi using a CC2531 USB though I do not know the revision of the firmware.  I have tried for days to pair the second unit.  It does pair and I can see it clearly in the configuration.yaml file.
If I look at the console log any action of the switch results in the following:
`Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'`

Does this suggest that the device is faulty, wasn't flashed properly or is this a familiar error?

Thanks in advance,

Garry

**Reply by lcx** [reply]
oops, seems like I didn't find this discussion and created a new one #10118
@GryKyo did you find a solution?
